pkgs/top-level/all-packages.nix

```
@@ -26248,6 +26248,8 @@ in
 
   armagetronad = callPackage ../games/armagetronad { };
 
+  armagetronad-dedicated = callPackage ../games/armagetronad { dedicatedServer = true; };
```

```suggestion
  armagetronad-server = callPackage ../games/armagetronad { dedicatedServer = true; };
```

or dedicated server which is a bit long.

---

Hmm, the compiled binary is called armagetronad-dedicated. I was going for consistency with that.

---

Then it makes sense to keep the name.